src/module/combat/combat-set-groups.ts

```diff
@@ -41,7 +41,7 @@ export default class OSECombatGroupSelector extends HandlebarsApplicationMixin(A
 
   static PARTS = {
     main: {
-      template: `/systems/ose-dev/dist/templates/apps/combat-set-groups.hbs`
+      template: `/systems/${game.system.id}/dist/templates/apps/combat-set-groups.hbs`
```

Comment:
This app finds the template when you do this, right? I think the reason I did this initially was `game.system.id` not being available when this component (but I forgot to swing back around to it and figure something out for it).

Reply:
No it doesn't! I thought for sure it would. Ok fixing this